### PR TITLE
Added product view

### PR DESCRIPTION
**Pull request by SzymonStolarski**
Hi,

I commited the frontend changes on adding the product view. @kasia-majchrzak i'd like to gently ask you for a CR before I can merge this.

s.

### CODE REVIEW

**Comment by SzymonStolarski** [comment]
@bentkowskip @Librist-pijo can I have one of you merge this once @kasia-majchrzak  approves the changes? Cheers.

**Review by kasia-majchrzak — APPROVED** [review]

**Pull request merged**
